model/docs/example-with-datasets.md

```diff
@@ -199,6 +199,10 @@ latent_hosp = latent.HospitalAdmissions(
     )
 ```
 
+    /mnt/c/Users/xrd4/Documents/repos/msr/model/.venv/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
+      from .autonotebook import tqdm as notebook_tqdm
+    An NVIDIA GPU may be present on this machine, but a CUDA-enabled jaxlib is not installed. Falling back to cpu.
```

Comment:
Can suppress this warning with `numpyro.set_platform("cpu")`

Reply:
Since I got your approval, I'll merge this and create an issue to add that line to the examples.